pkg/docker/docker_util.go

```diff
@@ -176,12 +142,21 @@ func ReadLogs(ctx context.Context, cli Docker, id string) (*bufio.Scanner, error
 func WaitForSandbox(reader *bufio.Scanner, message string) bool {
 	for reader.Scan() {
 		if strings.Contains(reader.Text(), message) {
+			kubeconfig := strings.Join([]string{
+				"$KUBECONFIG",
+				f.FilePathJoin(f.UserHomeDir(), ".kube", "config"),
+				Kubeconfig,
+			}, ":")
+			os.Setenv("KUBECONFIG", kubeconfig)
+			os.Setenv("FLYTECTL_CONFIG", util.FlytectlConfig)
+
```

Comment:
Whats the purpose of doing this when we asking the user to do it by echoing the export command